extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,11 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery-mask {
```

Comment:
I don't think we need a mask div here. We should be able to get away with a single div for the gallery, maybe call it `-amp-lightbox-gallery` instead of `-amp-lightbox-gallery-grid` and apply the z-index to that one. Now z-index list would be:

.-amp-lightbox-viewer-mask 2147483643
.amp-lightboxed  2147483644
-amp-lightbox-gallery 2147483645
.amp-lightbox-viewer-button 2147483646